.github/workflows/download_pipeline.yml

```diff
@@ -69,11 +69,3 @@ jobs:
 
       - name: Inspect download
         run: tree ./${{ env.REPOTITLE_LOWERCASE }}
-
-      - name: Run the downloaded pipeline (stub)
-        id: stub_run_pipeline
-        continue-on-error: true
-        env:
-          NXF_SINGULARITY_CACHEDIR: ./
-          NXF_SINGULARITY_HOME_MOUNT: true
-        run: nextflow run ./${{ env.REPOTITLE_LOWERCASE }}/$( sed 's/\W/_/g' <<< ${{ env.REPO_BRANCH }}) -stub -profile test,singularity --outdir ./results
```

Comment:
So now neither the stub run nor the regular run of the downloaded pipeline is tested? As a user of an offline cluster, it's very nice to know that `nf-core download` works smoothly. But if there's no space maybe there's nothing to do.

Reply:
Yeah, that's right. The downloaded pipeline is not tested anymore. I can make sure that the downloaded pipeline works locally before a release, but the github runners just don't have enough space for both the downloaded containers and the tests :(